### PR TITLE
XML structure sync

### LINKED ISSUE #2068: Evaluate eQIP form XML schema
As an eApp developer I need to review the data schema for the Army PSI-CoE so that I can create a roadmap for how best to take data from eApp and push it into the PSI-CoE format.

Acceptance Criteria
 - analyze provided XML schema
 - review data mappings and formulate a plan for migrating data

### PR DESCRIPTION
**Pull request by bmallred**
Resolves #2068 

Synchronization to the latest XML Schema Definition. Main sections with changes were:

 - Medical record
 - Citizenship
 - Foreign Activities

Added missing XML for foreign passports and foreign travel.

### CODE REVIEW

**Comment by floresj** [comment]
:tada:

File: api/model/form/templates/foreign-indirect-interests.xml [hunk 1]
```diff
@@ -16,52 +14,42 @@
       </LegalName>
       {{end}}
     </ControllingParty>
-
```
Comment (floresj) [inline api/model/form/templates/foreign-indirect-interests.xml]: Sorrrry I'm a sucker for spaces haha.

File: api/model/form/templates/spouse-former.xml [hunk 2]
```diff
@@ -25,6 +25,7 @@
       <Answer>{{branch $Item.Deceased}}</Answer>
     </Deceased>
     <DivorcedAnnulledComment></DivorcedAnnulledComment>
+    <DivorcedAnnulledComment2></DivorcedAnnulledComment2>
```
Comment (floresj) [inline api/model/form/templates/spouse-former.xml]: Why have 1 comment when you can have 2! :tada:

File: api/model/form/templates/spouse-present-marriage.xml [hunk 3]
```diff
@@ -5,6 +5,13 @@
     <APOFPO>
       <Address>{{location $Item.Address}}</Address>
       <AddressComment></AddressComment>
+      <!-- TODO -->
```
Comment (floresj) [inline api/model/form/templates/spouse-present-marriage.xml]: Is this one of those scenarios where the content guide may be different from that new schema?

File: src/components/Section/Relationships/Relatives/Relative.jsx [hunk 4]
```diff
@@ -404,10 +404,10 @@ export default class Relative extends ValidationElement {
                    onError={this.props.onError}
                    onUpdate={this.updateRelation}
                    />
-            <Radio name="relation-montherinlaw"
```
Comment (floresj) [inline src/components/Section/Relationships/Relatives/Relative.jsx]: Case of the **mon**days.

File: api/model/form/templates/spouse-former.xml (hunk 2 above)
Comment (bmallred) [reply api/model/form/templates/spouse-former.xml]: The first was annulled :drum:

File: api/model/form/templates/spouse-present-marriage.xml (hunk 3 above)
Comment (bmallred) [reply api/model/form/templates/spouse-present-marriage.xml]: This was a common pattern for a lot of their "APOFPO" addresses.

```xml
<APOFPO>
  <Address></Address>
  <AddressComment></AddressComment>
</APOFPO>
<Address></Address>
```

**Pull request merged**